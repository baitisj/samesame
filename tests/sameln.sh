NAME=sameln
. ${srcdir}/tests/include_action.sh
