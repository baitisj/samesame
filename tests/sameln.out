sameln -A:
./work/test1/file2a => ./work/test1/file1a
./work/test1/file3a => ./work/test1/file1a

sameln -Z:
./work/test1/file2a => ./work/test1/file3a
./work/test1/file1a => ./work/test1/file3a

sameln -At:
./work/test1/file3a => ./work/test1/file1a
./work/test1/file2a => ./work/test1/file1a

sameln -Zt:
./work/test1/file3a => ./work/test1/file2a
./work/test1/file1a => ./work/test1/file2a

sameln -L:
./work/test1/file1a => ./work/test1/file2a
./work/test1/file3a => ./work/test1/file2a
